store-kotlin/src/main/kotlin/com/nytimes/android/external/store3/base/impl/FluentMemoryPolicyBuilder.kt

```diff
@@ -0,0 +1,28 @@
+package com.nytimes.android.external.store3.base.impl
+
+/**
+ * Wraps methods for fluent MemoryPolicy instantiation.
+ */
+class FluentMemoryPolicyBuilder private constructor() {
```

Comment:
lets mark this class with @Experimental annotation. you can create that as a new marker interface. This way we can take a few minor revisions to stabilize api prior to shipping